libs/langchain-anthropic/src/chat_models.ts

```diff
@@ -100,6 +100,26 @@ function isAnthropicTool(tool: any): tool is Anthropic.Messages.Tool {
   return "input_schema" in tool;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function isBuiltinTool(
+  tool: any
```

Comment:
nit: use `unknown` - that's why we have the ESLint error

Reply:
Done!